nsis-semvercompare/src/lib.rs

```diff
@@ -33,6 +33,7 @@ fn semver_compare(v1: &str, v2: &str) -> i32 {
         (Ok(v1), Ok(v2)) => (v1, v2),
     };
 
+    #[allow(clippy::comparison_chain)]
```

Comment:
I didn't go for that because of the Known Problems section

> The match statement may be slower due to the compiler not inlining the call to cmp. See issue [#5354](https://github.com/rust-lang/rust-clippy/issues/5354)

But idk why since that doesn't actually matter here...